docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
         ports:
             - 80:3000
     db:
-        image: mcr.microsoft.com/mssql/server:2019-latest
+        image: mcr.microsoft.com/azure-sql-edge
```

Comment:
Hey, you should revert this change because we don't want to change the main docker